packages/blockchain-wallet-v4-frontend/src/modals/Settings/TradingLimits/template.success.tsx

```diff
@@ -0,0 +1,423 @@
+            size='16px'
+            weight={600}
+            style={{ marginLeft: '20px' }}
+          >
+            <FormattedMessage
+              id='modals.tradinglimits.send_crypto'
+              defaultMessage='Send Crypto'
+            />
+          </Text>
+        </IconWrapper>
+        {getItemBadgeStatus(currentTier, ITEMS.SEND)}
+      </ContentItem>
+
+      <ContentItem>
+        <Icon name='qr-code' color='blue600' size='20px' />
+        <Text
+          color='grey900'
+          size='16px'
+          weight={600}
+          style={{ marginLeft: '20px' }}
+        >
+          <FormattedMessage
+            id='modals.tradinglimits.receive_crypto'
+            defaultMessage='Receive Crypto'
+          />
+        </Text>
+        {getItemBadgeStatus(currentTier, ITEMS.RECEIVE)}
+      </ContentItem>
+      <ContentItem>
+        <Icon name='arrow-switch-thick' color='blue600' size='20px' />
+        <Text
+          color='grey900'
+          size='16px'
+          weight={600}
+          style={{ marginLeft: '20px' }}
+        >
+          <FormattedMessage
+            id='modals.tradinglimits.swap_crypto'
+            defaultMessage='Swap Crypto'
+          />
+        </Text>
+        {getItemBadgeStatus(currentTier, ITEMS.SWAP)}
+      </ContentItem>
+
+      <ContentItem>
+        <Icon name='usd' color='blue600' size='20px' />
+        <Text
+          color='grey900'
+          size='16px'
+          weight={600}
+          style={{ marginLeft: '20px' }}
+        >
+          <FormattedMessage
+            id='modals.tradinglimits.cash_accounts'
+            defaultMessage='Cash Accounts'
+          />
+        </Text>
+        {getItemBadgeStatus(currentTier, ITEMS.CASH_ACCOUNT)}
+      </ContentItem>
+      <ContentItem>
+        <Icon name='credit-card-sb' color='blue600' size='20px' />
+        <Text
+          color='grey900'
+          size='16px'
+          weight={600}
+          style={{ marginLeft: '20px' }}
+        >
+          <FormattedMessage
+            id='modals.tradinglimits.buy_with_a_card'
+            defaultMessage='Buy with a Card'
+          />
+        </Text>
+        {getItemBadgeStatus(currentTier, ITEMS.BUY_WITH_A_CARD)}
+      </ContentItem>
+      <ContentItem>
+        <Icon name='bank-filled' color='blue600' size='20px' />
+
+        <Text
+          color='grey900'
+          size='16px'
+          weight={600}
+          style={{ marginLeft: '20px' }}
+        >
+          <FormattedMessage
+            id='modals.tradinglimits.deposits_and_withdrawals'
+            defaultMessage='Deposits & Withdrawals'
+          />
+        </Text>
+        {getItemBadgeStatus(currentTier, ITEMS.DEPOSIT_AND_WITHDRAWAL)}
+      </ContentItem>
+      <ContentItem>
+        <Icon name='percentage' color='blue600' size='20px' />
+
+        <Text
+          color='grey900'
+          size='16px'
+          weight={600}
+          style={{ marginLeft: '20px' }}
+        >
```

Comment:
this exact same Text styling is repeated 7 times, please create a single styled-component